src/os/bluestore/BlueStore.h

```diff
@@ -1545,6 +1549,15 @@ class BlueStore : public ObjectStore,
     }
 #endif
 
+    inline void set_state(state_t s) {
```

Comment:
Suggest to make state private. And provide 'getter' to read it. Just to be absolutely sure that 'state' is properly encapsulated and accessed.